node/src/events.rs

```diff
@@ -167,6 +167,8 @@ pub struct NodeEventInfo {
 #[serde(tag = "type")]
 #[serde(rename_all = "snake_case")]
 pub enum NodeEvent {
+    ConnectingOnBootnodes,
```

Comment:
sounds more natural to me
```suggestion
    ConnectingToBootnodes,
```